libnavigation-core/src/test/java/com/mapbox/navigation/core/routeoptions/MapboxRouteOptionsUpdateCommonTest.kt

```diff
@@ -0,0 +1,49 @@
+package com.mapbox.navigation.core.routeoptions
+
+import com.mapbox.api.directions.v5.models.RouteOptions
+import org.junit.Assert
+
+object MapboxRouteOptionsUpdateCommonTest {
+
+    fun checkUnmutableFields(routeOptions: RouteOptions, updated: RouteOptions) {
+        Assert.assertEquals("Check Overview", routeOptions.overview(), updated.overview())
```

Comment:
```suggestion
       assertEquals("Check Overview", routeOptions.overview(), updated.overview())
```

Same thing applies to the rest of assertions 👇